src/main/java/org/team1540/robot2018/commands/auto/AutonomousProfiling.java

```diff
@@ -2,243 +2,71 @@
 
 import static org.team1540.robot2018.Tuning.timeStep;
 
-import edu.wpi.first.wpilibj.DriverStation;
 import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj.command.Command;
 import edu.wpi.first.wpilibj.command.Scheduler;
-import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
-import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import jaci.pathfinder.Pathfinder;
 import jaci.pathfinder.Trajectory;
 import jaci.pathfinder.Trajectory.Config;
 import jaci.pathfinder.Trajectory.FitMethod;
 import jaci.pathfinder.Trajectory.Segment;
 import jaci.pathfinder.Waypoint;
 import jaci.pathfinder.modifiers.TankModifier;
-import java.util.LinkedList;
-import java.util.List;
-import openrio.powerup.MatchData;
-import openrio.powerup.MatchData.GameFeature;
-import openrio.powerup.MatchData.OwnedSide;
 import org.team1540.base.motionprofiling.MotionProfilingProperties;
 import org.team1540.robot2018.Robot;
 import org.team1540.robot2018.Tuning;
 
 public class AutonomousProfiling extends Command {
 
-  // TODO Any changes based on the fact that the center of turning isn't in the center?
+  private Trajectory.FitMethod fitMethod = FitMethod.HERMITE_CUBIC;
 
-  private static final double BOT_BUMPER_WIDTH = 3.25;
-  private static final double BOT_LENGTH = 32.625;
-  private static final double BOT_EFFECTIVE_LENGTH = BOT_LENGTH + 2 * BOT_BUMPER_WIDTH;
-  private static final double BOT_SHORT_DISTANCE_TO_TURNING = 27.75;
-  private static final double BOT_WIDTH = 27.75;
-  private static final double BOT_EFFECTIVE_WIDTH = BOT_WIDTH + 2 * BOT_BUMPER_WIDTH;
+  private double maxVelocity = Tuning.maxVelocity;
```

Comment:
Use tuning values directly

Reply:
I wanted to set default values from tuning but allow the use of custom max velocities for different profiles, is there a better way of doing this?

Reply:
Ah, I see. Not really a better way of doing that then that isn't clunky as hell.